src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -28,7 +28,7 @@
 
 public class IntegrityCheck {
 
-    private final BibDatabaseContext bibDatabaseContext;
+    private static BibDatabaseContext bibDatabaseContext;
```

Comment:
>. An alternative is, to define a private final BibDatabaseContext
bibDatabaseContext; in class EditionChecker,

Seems to be the better solution as static variables sound like global
variables...